### PR TITLE
Counting Instance Segmentation - EfficientNetB2B FP16

### PR DESCRIPTION
**Pull request by eugene123tw**
add EfficientNetB2B FP16 backbone for counting instance segmentation

### CODE REVIEW

**Comment by eugene123tw** [comment]
1292

File: mmdet/apis/ote/extension/datasets/data_utils.py [hunk 1]
```diff
@@ -187,7 +188,7 @@ def _filter_imgs(self, min_size=32):
             img_id = self.img_ids[i]
             if self.filter_empty_gt and img_id not in ids_in_cat:
                 continue
-            if min(img_info["width"], img_info["height"]) >= min_size:
+            if min(img_info["width"], img_info["height"]) > self.min_size:
```
Comment (Ilya-Krylov) [inline mmdet/apis/ote/extension/datasets/data_utils.py]: self.min_size is not about image size, right?

File: mmdet/apis/ote/extension/datasets/data_utils.py [hunk 2]
```diff
@@ -85,7 +86,7 @@ def __init__(
         img_prefix="",
         test_mode=False,
         filter_empty_gt=True,
-        min_size=None,
+        min_size=32,
```
Comment (Ilya-Krylov) [inline mmdet/apis/ote/extension/datasets/data_utils.py]: Why do you change it here? Maybe it is better to change configuration file?

File: mmdet/datasets/custom.py [hunk 3]
```diff
@@ -162,14 +164,14 @@ def pre_pipeline(self, results):
         results['mask_fields'] = []
         results['seg_fields'] = []
 
-    def _filter_imgs(self, min_size=32):
+    def _filter_imgs(self):
```
Comment (Ilya-Krylov) [inline mmdet/datasets/custom.py]: why do you need to change IMAGE filtering?

File: tests/test_ote_api.py [hunk 4]
```diff
@@ -533,3 +533,9 @@ def test_training_maskrcnn_resnet50(self):
         self.end_to_end(osp.join('configs', 'ote',
                         'custom-counting-instance-seg', 'resnet50_maskrcnn'),
                         task_type=TaskType.INSTANCE_SEGMENTATION)
+
+    @e2e_pytest_api
+    def test_training_maskrcnn_efficientnetb2b(self):
+        self.end_to_end(osp.join('configs', 'ote',
+                        'custom-counting-instance-seg', 'efficientnetb2b_maskrcnn'),
+                        task_type=TaskType.INSTANCE_SEGMENTATION)
```
Comment (Ilya-Krylov) [inline tests/test_ote_api.py]: new line

File: mmdet/datasets/custom.py [hunk 5]
```diff
@@ -68,7 +68,8 @@ def __init__(self,
                  seg_prefix=None,
                  proposal_file=None,
                  test_mode=False,
-                 filter_empty_gt=True):
+                 filter_empty_gt=True,
+                 min_size=32):
```
Comment (druzhkov-paul) [inline mmdet/datasets/custom.py]: In which scenario CustomDataset is used directly? Or when these changes make any difference?

**Comment by eugene123tw** [comment]
@Ilya-Krylov @druzhkov-paul, I revert the changes of min_size as I thought it's used to filter annotations.

**Review by druzhkov-paul — APPROVED** [review]

**Review by Ilya-Krylov — APPROVED** [review]

**Pull request merged**